test/ring_self_intersection/test.cpg

```diff
@@ -0,0 +1 @@
+utf-8
```

Comment:
missing eol here and in other files. Is it on purpose ?

Reply:
Nope - these were generated from `ogr2ogr`. I'm not sure I understand fully how to format all of the files tbh.